src/lfx/src/lfx/components/elastic/opensearch.py

```diff
@@ -82,7 +82,8 @@ class OpenSearchVectorStoreComponent(LCVectorStoreComponent):
                 },
             ],
             value=[],
-            advanced=True,
+            # advanced=True,
```

Comment:
Unless there's a good reason, i'd say for readability lets just remove this line instead of commenting it out